client/web/web.go

```diff
@@ -487,6 +475,19 @@ func (s *Server) csrfKey() []byte {
 // Unlike http.StripPrefix, it does not return a 404 if the prefix is not present.
 // Instead, it returns a redirect to the prefix path.
 func enforcePrefix(prefix string, h http.HandlerFunc) http.HandlerFunc {
+	if prefix == "" {
+		return h
+	}
+
+	// ensure that prefix always has both a leading and trailing slash so
+	// that relative links for JS and CSS assets work correctly.
+	if prefix[0:1] != "/" {
```

Comment:
reads better if you use `strings.HasPrefix` and `strings.HasSuffix`

Reply:
🤦🏻 No idea why I didn't do that